lib/temporal/workflow/poller.rb

```diff
@@ -60,6 +60,11 @@ def poll_for_task
         client.poll_workflow_task_queue(namespace: namespace, task_queue: task_queue)
       rescue StandardError => error
         Temporal.logger.error("Unable to poll workflow task queue: #{error.inspect}")
+
+        Temporal.configuration.error_handlers.each do |handler|
```

Comment:
Not sure about this one… or maybe we need to have multiple handler types

Reply:
We need to be able to report WorkflowNotRegistered.

Reply:
This is required to catch things like an undefined workflow class. I don't really see a reason to treat this any differently. What's your concern here?